test/runtime/klibs.manifest

```diff
@@ -0,0 +1,25 @@
+(
+    boot:(
+          children:(
+                    kernel:(contents:(host:output/platform/pc/bin/kernel.img))
```

Comment:
Specifying the kernel image file here is redundant, since it's overwritten by mkfs when invoked with the -k option.